package.json

```diff
@@ -28,7 +28,7 @@
     "prepare": "husky",
     "lint-staged": "lint-staged --relative",
     "lint": "prettier . --check",
-    "format": "prettier . --write"
+    "format": "prettier . --write && pnpm dlx prisma format --schema apps/website/prisma/schema.prisma"
```

Comment:
I'm not sure I like the idea of calling a prisma command from the root package, rather than from the package where the file + dependency live. Can we move this into apps/website/package.json?

Reply:
Yeah I can move it to the website package.

Should It still be possible to call it from root though?

I think that having it as a script in the website package and running it in the pre-commit hook should be enough.

Reply:
pnpm has a recursive flag, so running the root format can run the format command in each workspace too

Reply:
I've moved it to the website package like you suggested

Reply:
I've moved it like you suggested